code/modules/admin/admin.dm

```diff
@@ -847,7 +847,9 @@ var/global/floorIsLava = 0
 	set desc="Start the round RIGHT NOW"
 	set name="Start Now"
 	if(GAME_STATE < RUNLEVEL_LOBBY)
-		alert("Unable to start the game as it is not set up.")
+		SSticker.auto_start = !SSticker.auto_start
+		message_admins(SPAN("info", "[key_name(src)] set the server start round configuration to [SSticker.auto_start ? "automatically start game as soon as possible" : "start game in normal mode (with a timer)."]"))
+		alert("Unable to start the game as it is not set up. [SSticker.auto_start ? "It will automatically start game as soon as possible" : "It will start game in normal mode (with a timer)."]")
```

Comment:
А может уберёшь этот надоедливый `alert()` и заменишь его на `to_chat()`?

```suggestion
		to_chat(owner, SPAN_NOTICE("Unable to start the game as it is not set up. [SSticker.auto_start ? "It will automatically start game as soon as possible" : "It will start game in normal mode (with a timer)."]"))
```

Reply:
Ну можно, это не было моей цели, но это лучше.